docs/docs/legal/acceptable-use-policy.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Acceptable Use Policy
+custom_edit_url: null
+--- 
+
+## Introduction
+
+This policy is an annex to the [Terms of Service](terms-of-service) and clarifies your obligations while using our Services.
+
+## Use
+
+You will ensure that the use of our Subscription Services and Website by yourself, your customers, or third parties comply with all applicable legislation.
+
+You may not: 
+1. Use Subscription Services or Website for phishing, social engineering, or committing fraud or any other illegal, malicious or fraudulent activity
+2. Attempt to interfere with the functionality or proper working of the Subscription Services or Website
+3. Upload any materials to the Subscription Services or Website in violation of any third-party privacy or data protection rights, to store and transmit any kind of malware
+4. Attempt to probe, scan, penetrate or test the vulnerability of our Subscription Services, Website, systems, or network or try to circumvent our authentication. Any penetration testing must not be conducted without prior written consent by CAOS. 
+5. Use any organization or domain name that includes or is confusingly similar with trademarks, or any third parties. CAOS may determine any violation at its sole discretion
+6. Collecting any information about our Customers, our Customers users, or our users without the consent of the person identified. This includes phishing, social engineering, scamming, spidering or harvesting information from any Subscription Service or Website
+
+## Fair Use Principle
+The “fair use” principle applies to the use of our services. We optimize our infrastructure in such a way that sufficient capacity is available to you even during short-term increased demand (“peaks”) and implement mitigation measures, implementing our [Rate Limit Policy](rate-limit-policy).  You are nonetheless required to adhere to reasonable use of our resources in order to avoid negatively affecting the services for other customers.
```

Comment:
```suggestion
The “fair use” principle applies to the use of our services. We optimize our infrastructure in such a way that sufficient capacity is available to you even during short-term increased demand (“peaks”) and implement mitigation measures such as our [Rate Limit Policy](rate-limit-policy).  You are nonetheless required to adhere to reasonable use of our resources in order to avoid negatively affecting the services for other customers.
```